homeassistant/components/sensor/zwave.py

```diff
@@ -33,26 +34,36 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     # if 1 in groups and (NETWORK.controller.node_id not in
     #                     groups[1].associations):
     #     node.groups[1].add_association(NETWORK.controller.node_id)
+    name = '{}.{}'.format(DOMAIN, zwave.object_id(value))
+    node_config = customize.get_overrides(hass, zwave.DOMAIN, name)
```

Comment:
force_update should be triggered by something on a device. We should be able to detect if a device has new information or is rebroadcasting existing state when we receive a value.

(btw, we should really rename `customize` to `configuration` because it has a completely different intent then `customize` in `homeassistant` config)